tests/rpc/test_rpc.py

```diff
@@ -26,7 +26,8 @@ def prec_satoshi(a, b) -> float:
 
 
 # Unit tests
-def test_rpc_trade_status(default_conf, ticker, fee, markets, mocker) -> None:
+@pytest.mark.asyncio
+async def test_rpc_trade_status(default_conf, ticker, fee, markets, mocker) -> None:
```

Comment:
This runs the code of the test within a coroutine - which is now how our code is running normally.

While i'm not 100% certain if that would be a problem, there is an easier (safer?) fix than that
for this test, and probably for the other tests, too, by replacing the first few lines with the following:

```python
    mocker.patch('freqtrade.rpc.telegram.Telegram', MagicMock())
    mocker.patch.multiple(
        'freqtrade.exchange.Exchange',
        get_ticker=ticker,
        get_fee=fee,
    )
    patch_exchange(mocker)
```

that'll remove the async warning, which is shown from within `_init_ccxt()` (that function is mocked completely).

I don't quite understand what pytest does differently so this happens (while it doesn't during regular operations) - but i think this should be the fix.

This method should be applicable to all tests (apart from the one testing `_init_ccxt()` - but i think that has this covered differently.